src/Equations/RunawayFluid.cpp

```diff
@@ -866,6 +866,28 @@ real_t* RunawayFluid::evaluatePartialContributionAvalancheGrowthRate(len_t deriv
     return dGamma;
 }
 
+/**
+ * Same as the function above, but for the formula published by Hesslow et al NF 2019
+ */
+real_t* RunawayFluid::evaluatePartialContributionAvalancheGrowthRateAlt(len_t derivId) {
+    real_t *dGamma = new real_t[nr];
```

Comment:
This is more of a comment to @Embreus, I guess, but I think we should try to **not** allocate memory in these functions. Because I suspect that these are called on every jacobian rebuild? In which case we include a bunch of expensive memory allocations, which also risk causing memory leaks.

Reply:
Indeed, I agree that this should be revised.